mesonbuild/compilers/fortran.py

```diff
@@ -430,7 +430,60 @@ def __init__(self, exelist: T.List[str], version: str, for_machine: MachineChoic
                           'everything': default_warn_args + ['-Mdclchk']}
 
 
-class FlangFortranCompiler(ClangCompiler, FortranCompiler):
+class LlvmFlangFortranCompiler(ClangCompiler, FortranCompiler):
+
+    id = 'llvm-flang'
```

Comment:
I guess I had already set a compiler-id here. Seeing how there's already `armltdflang`, I think adding the `flang` in the back rather than in the front might be better after all?